iree/vm/BUILD

```diff
@@ -183,9 +182,9 @@ cc_library(
     ],
     hdrs = [
         "bytecode_module.h",
+        "bytecode_op_table.h",
```

Comment:
run bazel_to_cmake to reflect these changes in the CMakeLists.txt file in this directory:

```bash
python3 build_tools/bazel_to_cmake/bazel_to_cmake.py
```

Reply:
Done